server/service/src/programs/patient/search.rs

```diff
@@ -37,16 +37,16 @@ pub fn patient_search(
     } = input;
 
     if let Some(code) = code {
-        filter = filter.code(SimpleStringFilter::equal_to(&code));
```

Comment:
@clemens-msupply, previously insensitive_equal_to would be the same as like but sanitised (with '%' removed), was there a danger that user would be searching with '%' ? Security risk or too many matches ?